### ISSUE TITLE
Inserting Float type requires a decimal value (different from 0)

### ISSUE THREAD

**Issue by arn00s** [prisma/prisma2#198]
Here's a sample of my model : 

```
model Reward {
  id        String     @default(cuid()) @id 
  value     Float
}
```

When trying to insert a value with this script

```
  //Rewards creation
  const reward1 = await photon.rewards.create({
    data: {
      value: 10.0
    }
  })
```

I get the following error : 

```
(node:9911) UnhandledPromiseRejectionWarning: Error: Error in Photon: 
ConnectorError(QueryError(Error { kind: ToSql(5), cause: Some(WrongType(Type(Numeric))) }

stack backtrace:
   0: backtrace::backtrace::trace::h5511721bea6fa8d5 (0x10478ca2e)
   1: backtrace::capture::Backtrace::new_unresolved::h28f9c9663a9420c8 (0x10478bb88)
   2: failure::backtrace::internal::InternalBacktrace::new::hbb41405b6d19ab81 (0x10478b4f9)
   3: <failure::backtrace::Backtrace as core::default::Default>::default::h0458ad281d4247c8 (0x10478b6e5)
   4: <sql_connector::error::SqlError as core::convert::From<tokio_postgres::error::Error>>::from::h4d6632d810ab6f89 (0x1040b5e9c)
   5: sql_connector::database::postgresql::<impl sql_connector::transactional::Transaction for postgres::transaction::Transaction>::write::ha2ca1483a41de8e3 (0x10408bf99)
   6: sql_connector::transactional::Transaction::insert::h102a04439344ffff (0x103c9c924)
   7: sql_connector::transactional::database_writer::create::execute::h2c63144580998c8d (0x1040ca331)
   8: sql_connector::transactional::database_writer::<impl connector::database_writer::DatabaseWriter for sql_connector::database::SqlDatabase<T>>::execute::{{closure}}::create::hef864a06474fbea9 (0x104107b47)
   9: <sql_connector::database::postgresql::PostgreSql as sql_connector::transactional::Transactional>::with_transaction::h9fe1ec452d2d39e2 (0x103d99d98)
  10: sql_connector::transactional::database_writer::<impl connector::database_writer::DatabaseWriter for sql_connector::database::SqlDatabase<T>>::execute::h292b3294dffefd85 (0x103d1e496)
  11: core::executor::write::WriteQueryExecutor::exec_one::h48f2e807cc785823 (0x1041b8146)
  12: core::executor::write::WriteQueryExecutor::execute::h992b6a51be508cce (0x1041b7a9b)
  13: core::executor::Executor::exec_all::h5cbc538ae21b3c80 (0x1041d6be4)
  14: <prisma::req_handlers::graphql::GraphQlRequestHandler as prisma::req_handlers::RequestHandler>::handle::h7d76032243e11158 (0x103e1eb3f)
  15: prisma::http_handler::h5782a9905407eb5f (0x103db94cf)
  16: <F as actix_web::with::FnWith<T,R>>::call_with::haa4a1a2255bc8611 (0x103e167d3)
  17: <actix_web::with::WithHandlerFut<T,S,R> as futures::future::Future>::poll::hd3a8444797502a6e (0x103e1cb52)
  18: actix_web::pipeline::PipelineState<S,H>::poll::h147d5d5368789f0e (0x103cf6c46)
  19: <actix_web::pipeline::Pipeline<S,H> as actix_web::server::handler::HttpHandlerTask>::poll_io::h39e2cc76fc5e8dd4 (0x103cea110)
  20: actix_web::server::h1::Http1Dispatcher<T,H>::poll_handler::hf7d43112cc4ed6f5 (0x103cbff4a)
  21: actix_web::server::h1::Http1Dispatcher<T,H>::poll::h9c1da5bf8f5108dd (0x103d8f1a3)
  22: <actix_web::server::channel::HttpChannel<T,H> as futures::future::Future>::poll::had9f1b0a81e71dda (0x103d8e079)
  23: <actix_web::server::channel::HttpChannel<T,H> as futures::future::Future>::poll::had9f1b0a81e71dda (0x103d8ef33)
  24: <actix_net::service::and_then::AndThenFuture<A,B> as futures::future::Future>::poll::h15ac475c98adc942 (0x103ce8a06)
  25: futures::future::chain::Chain<A,B,C>::poll::hb6e62549216884f5 (0x103ce4c84)
  26: futures::task_impl::std::set::he4a7d579f87ab972 (0x103f006fd)
  27: futures::task_impl::Spawn<T>::poll_future_notify::h1a5228822adb903e (0x103f1970f)
  28: tokio_current_thread::CurrentRunner::set_spawn::h6965d5b16e375442 (0x103efa674)
  29: tokio_current_thread::scheduler::Scheduler<U>::tick::h6c1934358ae3f6a8 (0x103f06a05)
  30: tokio_current_thread::Entered<P>::block_on::he3da6e046995d1e4 (0x103efb22a)
  31: std::thread::local::LocalKey<T>::with::h6ed9ac0420a84d3b (0x103f11904)
  32: std::thread::local::LocalKey<T>::with::h71f7d0c12da0393f (0x103f11e47)
  33: std::thread::local::LocalKey<T>::with::h10c82d1197b9368d (0x103f0fb2c)
  34: std::thread::local::LocalKey<T>::with::h38bd3e3c53efd4ea (0x103f11008)
  35: tokio::runtime::current_thread::runtime::Runtime::block_on::hbad87c088a1c5a72 (0x103efea92)
  36: std::sys_common::backtrace::__rust_begin_short_backtrace::h690508ce542785d7 (0x103f172a8)
  37: std::panicking::try::do_call::h1536644c0a3cab79 (0x103efcdfb)
  38: __rust_maybe_catch_panic (0x1047bd51f)
  39: core::ops::function::FnOnce::call_once{{vtable.shim}}::h3508efbc713fc3ba (0x103f17b36)
  40: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hf8766009c029bc09 (0x1047a471e)
  41: std::sys::unix::thread::Thread::new::thread_start::hfdef4649a42cf26c (0x1047bcb1e)
  42: _pthread_body (0x7fff596fd2eb)
  43: _pthread_start (0x7fff59700249)))
    at PhotonFetcher.<anonymous> (/node_modules/@generated/photon/index.js:33:23)
    at Generator.throw (<anonymous>)
    at rejected (/node_modules/@generated/photon/index.js:5:65)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
(node:9911) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:9911) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

Executing the same code with 

```
  //Rewards creation
  const reward1 = await photon.rewards.create({
    data: {
      value: 10.1
    }
  })
```

works perfectly fine.

**Comment by matthewmueller** [comment]
This looks like a serialization error, where it gets encoded as an integer when it should be a float.

**Comment by timsuchanek** [comment]
Duplicate of https://github.com/prisma/prisma2/issues/147

**Comment by arn00s** [comment]
Closed, as #147 was closed too.
